docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -0,0 +1,300 @@
+   ``parallel``.
+
+   .. code-block:: ini
+
+      scheduler = 'parallel'
+
+#. (Optional) Set the network key to specify secured network communication
+   between nodes in the network. By default, the network is unsecured.
+
+      .. important::
+
+         The example configuration file contains generic keys. You **must**
+         change these keys in order to have a secured network.
+
+   a. Locate the ``network_public_key`` and ``network_private_key`` settings.
+      These items specify the curve ZMQ key pair used to create a secured
+      network based on side-band sharing of a single network key pair to all
+      participating nodes.
+
+   #. Generate your network keys. The following example shows how to use
+      Python to generate these keys:
+
+         .. code-block:: python
+
+            python
+            .
+            .
+            .
+            >>> import zmq
+            >>> (public, prive) = zmq.curve_keypair()
+            >>> print public
+            ...
+            >>> print prive
+            ...
+
+   #. Replace the sample values with your network keys.
+
+      .. code-block:: ini
+
+         network_public_key = '{nw-public-key}'
+         network_private_key = '{nw-private-key}'
+
+#. After saving your changes, restrict the permissions on ``validator.toml``
+   to protect the network private key.
+
+   .. code-block:: console
+
+      $ sudo chown root:sawtooth /etc/sawtooth/validator.toml
+      $ sudo chown 640 /etc/sawtooth/validator.toml
+
+#. Finally, restart the validator to activate the configuration changes.
+
+   .. code-block:: console
+
+      $ sudo systemctl restart sawtooth-validator.service
+
+.. note::
+
+   To learn how to use the ``[role]`` and ``[permissions]`` settings to
+   control validator and user access to the network, see
+   :doc:`configuring_permissions`.
+
+   For information about the ``opentsdb_`` settings, see
+   :doc:`grafana_configuration`.
+
+
+.. _rest-api-bind-address-label:
+
+Configure the REST API
+=======================
+
+Use these steps to change the network settings for the REST API.
+
+#. Create the REST API configuration file by copying the example file.
+
+   .. code-block:: console
+
+      $ sudo cp /etc/sawtooth/rest_api.toml.example /etc/sawtooth/rest_api.toml
+
+#. Use ``sudo`` to edit this file.
+
+   .. code-block:: console
+
+      $ sudo vi /etc/sawtooth/rest_api.toml
+
+#. If necessary, change the ``bind`` setting to specify where the REST API
+   listens for incoming communication.
+
+   Be sure to remove the ``#`` comment character to activate this setting.
+
+   .. code-block:: console
+
+      bind = ["127.0.0.1:8008"]
+
+
+#. If necessary, change the ``connect`` setting, which specifies where the
+   REST API can find this node's validator. This setting must match the
+   ``bind = [... component`` value in the validator configuration file (see
+   :ref:`change-nw-validator-label`).
```

Comment:
This is slightly more complicated and matching the string passed to the validator will not always work. If you pass a device to the validator to bind to, such as `tcp://eth0:4004`, this will need to be the public endpoint for that device on the network shared by the REST API and Validator. For example, in this compose file we bind to the eth0 device for the validator's component endpoint, but connect with the validator's name on the network "validator".

https://github.com/hyperledger/sawtooth-raft/blob/master/adhoc/node.yaml#L67
https://github.com/hyperledger/sawtooth-raft/blob/master/adhoc/node.yaml#L109